js-exercises/.eslintrc

```diff
@@ -6,6 +6,7 @@
   "rules": {
     "no-restricted-syntax": 0,
     "arrow-parens": 0,
-    "import/prefer-default-export": 0
+    "import/prefer-default-export": 0,
+    "linebreak-style": 0
```

Comment:
As mentioned in the guidelines before, if you are making changes to ESlint or something which is directly not related to the question, please create a separate PR for that.

Reply:
right, will take care further